Sebastian-Sole/handicappin
# ✨ feat: theme

## What was done
- Updated the color theme of the website to improve user experience
- Updated components styling to improve how they look
- Improved continuity in design across components

## How to test
- Visit any page
- See that the page is blue and not green

## Future work
- None
